docs/source/conf.py

```diff
@@ -366,13 +367,14 @@ def generate_classification_table():
 
     generated_dir = Path("generated")
     generated_dir.mkdir(exist_ok=True)
-    with open(generated_dir / "classification_table.rst", "w+") as table_file:
+    with open(generated_dir / f"{table_name}_table.rst", "w+") as table_file:
         table_file.write(".. table::\n")
-        table_file.write("    :widths: 100 10 10 20 10\n\n")
+        table_file.write(f"    :widths: 100 {'15 ' * len(metrics_names)} 20 10\n\n")
         table_file.write(f"{textwrap.indent(table, ' ' * 4)}\n\n")
 
 
-generate_classification_table()
+generate_weights_table(module=M, table_name="classification", metrics_names=["Acc@1", "Acc@5"])
+generate_weights_table(module=M.detection, table_name="detection", metrics_names=["box_map"])
```

Comment:
I think this is going to put the KeypointRCNN model along with the rest of the models. Unfortunately the specific model was trained with different data from COCO (keypoints), so its focus and validation numbers are the same as for other models. 

Have a look on the current documentation [here](https://github.com/pytorch/vision/blob/main/docs/source/models.rst#object-detection-instance-segmentation-and-person-keypoint-detection) to see how the models are grouped and how extra accuracy metrics are being shown. We don't have to do exactly the same thing but good to get some ideas.

Reply:
Is it OK with you if I create an issue to special-case KeypointRCNN? This way I'll still be able to ask for external contributors while I'm thinking of a decent solution for this.

Reply:
Yeah I think that would be fine.

Reply:
Yeah I think that would be fine, let's do this.